packages/app/cypress/e2e/cypress-in-cypress-e2e.cy.ts

```diff
@@ -151,7 +151,7 @@ describe('Cypress In Cypress E2E', { viewportWidth: 1500, defaultCommandTimeout:
   })
 
   it('shows a compilation error with a malformed spec', { viewportHeight: 596, viewportWidth: 1000 }, () => {
-    const expectedAutHeight = 456 // based on explicitly setting viewport in this test to 596
+    const expectedAutHeight = 500 // based on explicitly setting viewport in this test to 596
```

Comment:
This has legitimately changed, previously the browser dropdown at this width was lessening the AUT height. 
![Screenshot 2025-03-05 at 12 10 26 PM](https://github.com/user-attachments/assets/2223d76a-0db6-4208-af22-3afd4c591dd7)

![Screenshot 2025-03-05 at 12 10 37 PM](https://github.com/user-attachments/assets/1e81f265-de08-429c-a59f-4edfd671e09a)